File: lista 2/5.py
r1, r2, r3, r4, r5 = map(int, input().split())

medicoes = [r1, r2, r3, r4, r5]
media = sum(medicoes) / 5.0

if media < 300:
    print("Ambiente seguro")
elif media <= 600:
    print("Ambiente instavel")
else:
    print("Ambiente altamente perigoso")

if any(r > 600 for r in medicoes):
    print("Pelo menos um local altamente perigoso")

File: lista 2/5.cpp
#include <iostream>

using namespace std;

int main() {
    int r1, r2, r3, r4, r5;
    cin >> r1 >> r2 >> r3 >> r4 >> r5;

    double media = (r1 + r2 + r3 + r4 + r5) / 5.0;

    if (media < 300) {
        cout << "Ambiente seguro" << endl;
    } else if (media <= 600) {
        cout << "Ambiente instavel" << endl;
    } else {
        cout << "Ambiente altamente perigoso" << endl;
    }

    if (r1 > 600 || r2 > 600 || r3 > 600 || r4 > 600 || r5 > 600) {
        cout << "Pelo menos um local altamente perigoso" << endl;
    }

    return 0;
}